templates/generic/rpi4_64_reference_bb_kirkstone.json

```diff
@@ -0,0 +1,200 @@
+{
+    "platformName": "rpi4_64_reference",
+    "os": "linux",
+    "arch": {
+        "arch": "aarch64",
+        "variant": "v8"
+    },
+    "rdk": {
+        "version": "2024Q3",
+        "supportedFeatures": [
+            "Controller",
+            "DeviceInfo",
+            "WifiManager",
+            "com.comcast.DeviceProvisioning",
+            "org.rdk.DeviceDiagnostics",
```

Comment:
keep things which are applicable for broadband,still few changes are related to video

Reply:
Addressed Chandra.
Please review